spec/models/reports/repository_spec.rb

```diff
@@ -1,623 +1,590 @@
 require "rails_helper"
 
-RSpec.describe Reports::Repository, type: :model, v2_flag: true do
```

Comment:
This diff is only large because of whitespace - viewing with whitespace ignored will help.